docs/sop/PRODUCTION_DEPLOY.md

```diff
@@ -0,0 +1,344 @@
+
+The terraform state file is stored remotely in gitlab using terraform `http` backend module,
+details of the implementation can be found at [here](https://docs.gitlab.com/ee/user/infrastructure/iac/terraform_state.html).
+As a result, the state file can be shared which allows multiple developers to provision the same existing infrastructure.
+One important thing to note is that the existing infrastructure can only be provisioned by 1 developer at a time, as the remote state file
+will be locked, this help to make sure that each developer only works on the most recent infrastructure.
+
+If you have any problem during the deployment process, please first check the [troubleshooting guide](PRODUCTION_TROUBLESHOOT.md).
+
+## Prerequisites
+
+### AWS IAM policies
+[Click to see how to check IAM policies](AWS_SETUP.md#verify-aws-iam-policies)
+
+### Elastic IP addresses for staging and live environments
+
+Both the staging.gigadb.org and gigadb.org domain names have been allocated
+with AWS elastic IP addresses in the ap-east-1 region. You can check the current 
+elastic IP addresses pointing in the AWS EC2 console:
+
+1. Go to the [cnhk-infra CI/CD variables page](https://gitlab.com/gigascience/cnhk-infra/-/settings/ci_cd) 
+to get the password for the `Gigadb` AWS IAM user.
+2. Use the `Gigadb` AWS IAM user credentials to log into the AWS console.
+3. Go to the [Elastic IP addresses page](https://ap-east-1.console.aws.amazon.com/ec2/v2/home?region=ap-east-1#Addresses:)
+4. Check there is an `eip-gigadb-live-gigadb` and an `eip-gigadb-staging-gigadb` 
+elastic IP address.
+
+### Domain name resolution to staging.gigadb.org and gigadb.org
+
+Resolution to staging and live gigadb.org with the above elastic IP addresses 
+require DNS A records. Check these records have been created in Alibaba Cloud 
+console:
+
+1. Go to the [cnhk-infra CI/CD variables page](https://gitlab.com/gigascience/cnhk-infra/-/settings/ci_cd)
+to get the `Alibaba_user_email` and `Alibaba_user_password`.
+2. Log into the [Alibaba Cloud console](https://account.alibabacloud.com/login/login.htm?oauth_callback=https%3A%2F%2Fhome-intl.console.aliyun.com%2F%3Fspm%3Da3c0i.7911826.6791778070.41.44193870AxVzyk&lang=en) using the above credentials.
+3. You will be asked for a 6 digit one time password (OTP) that is provided by 
+the Google Authenticator app.
+> :warning: **You'll need to contact pli888 for the OTP**
+4. Once logged into the console, go to the Domain Names page
+5. You will see an entry for `gigadb.org` domain - click on this
+6. You will now see the `DNS Settings gigadb.org` page. There should be an entry
+for the Host with a value equal to the `eip-gigadb-live-gigadb` elastic 
+IP address. There will also be an entry for the `staging` Host too.
+
+### Set up credentials for accessing AWS resources
+[Click to see how to setup credentials](AWS_SETUP.md#set-up-credentials-for-accessing-aws-resources)
+
+### Tools
```

Comment:
As I found out from trying to access AWS resources in a new Gigadb user account last week, other tools like Terraform, Ansible, terraform-inventory, AWS CLI, jq are required too. We take the fact that these tools have already been installed from out developer work.